### ISSUE TITLE
Don't you consider making a colab notebook for your tool?

### ISSUE THREAD

**Issue by vlad-dostavalov** [eps696/stylegan2#4]
Thank your for sharing optimised for artistic purposes stylegan2 fork! Great job.
This is what I was looking for. If it also had the ability to make audio reactive videos if would be amazing.

So my question is: Don't you consider making a colab notebook for your tool?
Many people like me use colab for training and cannot afford to conduct computations on home pc or on a windows server.
Sorry for stupid questions, but I as an artist, not a programmer, find it difficult to run your code on Colab.

**Comment by eps696** [comment]
thank you for your interest. 
i do use various python-based neural networks for realtime performance and/or installations, using custom OSC and [Spout](https://github.com/spiraltechnica/Spout-for-Python) integrations. I also consider publishing my modules and scripts for such integrations, but not quite soon (it needs some polishing yet, and maybe proper workshop on the topic). 
as for colab, i doubt i can help with that now, as i'm not experienced in it (never used it for production, and tbh not aware of its realtime usability at all).

**Comment by eps696** [comment]
@vlad-dostavalov if it's relevant yet, i'll be doing some colab-based class in february 2021, so will have this repo adapted for colab use in january (probably with all new tricks which i find useful for that moment)

**Comment by vlad-dostavalov** [comment]
Cool! Please let me know when it's gonna happen, I'm very interested in new tools. By the way, you said you were working on realtime performances. Btw in March-April we are going to conduct a multidisciplinary festival, called WeCosmos, with light/video shows, installations, etc. Just thought you could be interested

**Comment by eps696** [comment]
@vlad-dostavalov sure interested, drop me a line on eps (at) here.ru or eps696 on telegram/fb/twitter/..

**Comment by eps696** [comment]
as for audio-reactive GAN visuals, here are few nice approaches (grabbed from the recent NeurIPS Creativity Workshop):
https://github.com/JCBrouwer/maua-stylegan2
https://jdasam.github.io/traeumerAI_demo
https://psc-g.github.io/posts/research/creativity/ganterpretations
(more interesting links [here](https://docs.google.com/document/d/1TXIROUYYZtIZyHCIql6ihDrkt7q_lMUIGx90cBvTe7A))

**Comment by Thierryonre** [comment]
I've made a colab where I've set it up but I'm running into OOM problems.
I first tried solving it by using config E but that didn't work so I halved the resolution of my images from 2000x2828 to 1000x1414 (which are still quite large). This is with 488 images as well but with the halving resolution, it got to the point where it said "Building TensorFlow graph..." which is further than the time before.
So I'll have the size of my images again and see what happens.
I might then add a resolution enhancer into the colab to get them to the size I want :D
The images are also further reduced in size to an acceptable 2^n resolution.
